client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsTabs.tsx

```diff
@@ -48,18 +32,24 @@ const ContactsTabs = ({ isVisible }: { isVisible: boolean }) => {
                 value={currentTab}
                 onChange={handleChange}
             >
-                <Tab classes={tabClasses}
-                    icon={<FontAwesomeIcon color={active} icon={faUserEdit} />}
-                    label='הוספת מגע ידנית' />
-                <Tab
-                    classes={tabClasses}
-                    icon={<GroupOutlined color='action' />}
-                    label='בני משפחה' />
+                <Tab classes={tabClasses} icon={<FontAwesomeIcon color={active} icon={faUserEdit} />} label='הוספת מגע ידנית' />
```

Comment:
maybe use const for labels

Reply:
moved [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1233/commits/e843da2788992ee4e113d24fb2f7d73c6386c548)